CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+### IN DEVELOPMENT
+
+#### User changes
+
+* You can now access the functional regular axis directly, `regular_sqrt` becomes `regular.sqrt`, etc.. [#183][]
```

Comment:
No `regular.sqrt`. The transform should be a passed as a keyword to the regular class.